content/blog/0000-00-00-dummy.md

```diff
@@ -0,0 +1,6 @@
+---
+title: dummy
```

Comment:
Small inclusivity request: `dummy` is a word with roots in ableism. If we could find a different word, that'd be ideal.

Reply:
How does the word `mock` sound? @bnb